lib/extensions/permissions.js

```diff
@@ -12,6 +13,17 @@ const STATUS = Object.freeze({
 });
 
 const WIX_SIM_UTILS = 'applesimutils';
+
+// `location` permission does not work with WIX/applesimutils.
```

Comment:
perhaps, we should also report this as an issue to Wix?

Reply:
Found https://github.com/wix/AppleSimulatorUtils/issues/111 in their repo. Let me link here